gum/backend-arm/gumstalker-arm.c

```diff
@@ -305,6 +308,9 @@ static gboolean gum_stalker_iterator_arm_next (GumStalkerIterator * self,
     const cs_insn ** insn);
 static gboolean gum_stalker_iterator_thumb_next (GumStalkerIterator * self,
     const cs_insn ** insn);
+static gboolean gum_is_exclusive_store_insn (const cs_insn * insn);
+static void gum_increment_exclusive_load_offset (GumGeneratorContext * gc);
```

Comment:
This is a method on the `Gum.GeneratorContext`, so the name should reflect that:

`gum_generator_context_advance_exclusive_load_offset`

Methods on that class should also be moved to their own group. (Groups are separated by a blank line up here.) The ordering of the groups should follow the order of the struct typedefs. (Which flow from highest to lowest level.)